jupyterhub/files/hub/cull_idle_servers.py

```diff
@@ -302,7 +318,11 @@ def handle_user(user):
            help="The maximum age (in seconds) of servers that should be culled even if they are active")
     define('cull_users', default=False,
            help="""Cull users in addition to servers.
-                This is for use in temporary-user cases such as tmpnb.""",
+                This is for use in temporary-user cases such as BinderHub.""",
+           )
+    define('remove_named_servers', default=False,
+           help="""Remove named servers in addition to stopping them.
+                This is for use in temporary-user cases such as Binderhub.""",
```

Comment:
These are authenticated not temporary users.

Will this also remove the default server? If not is there a way to enable that?

Reply:
I think your default server gets removed completely already by the standard culling. As I understand it your named servers as slightly different in that they don't vanish completely if you stop them. You can still see them in the dashboard and start them again.

Updated the help text.